engine/src/main/java/com/datatorrent/stram/webapp/TypeGraph.java

```diff
@@ -602,6 +558,14 @@ public void setJarName(String jarName)
 
     public CompactClassNode getClassNode()
     {
+      if (classNode == null) {
```

Comment:
Do we need double-checked locking here? Very slim chance, but is it likely that describeOperator  gets called simultaneously through dtcli?